src/app/examples/list/no-shape/list-no-shape-example.component.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../../kirby/scss/utils";
+
+kirby-card {
+  margin: map-get($shadow-offset-y, 2);
```

Comment:
@jakobe I'm using the variables as requested now. Luckily, the shadow-offset can be used here, but in reality the margin should be based on a calculation I guess, depending on what side the margin is on.
However I would like this not to be in scope of this PR, as I don't see it as a trivial thing to implement.